### DISCUSSION TITLE
Have a question about testing fscrawler with zip files

### DISCUSSION THREAD

**Discussion by abeaudin** [dadoonet/fscrawler#1764] [Q&A] [answered]
I think fscrawler is able to extract zipped files - I have a zip file with 4 txt files in it and I get a doc in my index for the zip file but it doesn't have anything in it wrt the contents of the zipfile - is there a flag I need to set to see the contents of the zip file?

eg
`vertige:~/fscrawler-distribution-2.10-SNAPSHOT# ls /home/user/dir/*.zip
/home/user/dir/file.zip
vertige:~/fscrawler-distribution-2.10-SNAPSHOT# unzip /home/user/dir/file.zip
Archive:  /home/user/dir/file.zip
 extracting: file1.txt               
 extracting: file2.txt               
 extracting: file3.txt               
 extracting: file4.txt               
`

yet in my index I only have the zip file and the listing of the files is in the record but not the content of those files
`\nfile1.txt\n\n\nfile2.txt\n\n\nfile3.txt\n\n\nfile4.txt\n\n`

Is there a way to extract the files as well?

I looked in the docs and didn't see anything related to file extraction from a zip file.

**Comment by abeaudin** [comment, marked as answer]
I've answered my own question - my files - file1.txt thru file4.txt were empty - I retested with the string one in 1file.txt thru 4file.txt and zipped them into num.zip and I can see the content now
`"\n1file.txt\none\n\n\n\n\n2file.txt\none\n\n\n\n\n3file.txt\none\n\n\n\n\n4file.txt\none\n\n\n\n"`

**Reply by dadoonet** [reply]
Amazing! Thanks for sharing.
